### PR TITLE
fix(playwright): Enable coppa and fix pw reset tests

### PR DESCRIPTION
**Pull request by vbudhram**
## Because

- I think I found the unlying issue with sync coppa test

## This pull request

- Cleans up password reset tests and enable coppa
- Combines pw validation and password reset into same test
- Removes creating duplicate accounts for the test

## Issue that this pull request solves

Closes: https://mozilla-hub.atlassian.net/browse/FXA-6936

## Checklist

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/functional-tests/tests/syncV3/signIn.spec.ts [hunk 1]
```diff
@@ -17,21 +17,13 @@ test.describe('Firefox Desktop Sync v3 sign in', () => {
   test('verified, does not need to confirm', async ({ target }) => {
     const { page, login, connectAnotherDevice, signinTokenCode } =
       await newPagesForSync(target);
-    const uaStrings = {
```
Comment (vbudhram) [inline packages/functional-tests/tests/syncV3/signIn.spec.ts:20]: I honestly don't know what this tests but it is over 50 versions old. I think it is safe to remove...

File: packages/functional-tests/tests/syncV3/signUp.spec.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ test.describe('Firefox Desktop Sync v3 sign up', () => {
     expect(await connectAnotherDevice.fxaConnected.isVisible()).toBeTruthy();
   });
 
-  test.skip('coppa disabled', async ({ target }) => {
```
Comment (vbudhram) [inline packages/functional-tests/tests/syncV3/signUp.spec.ts:46]: I'm not sure when we don't display coppa for sync user signups? Feels like something we can always show? @LZoog

Comment (LZoog) [reply packages/functional-tests/tests/syncV3/signUp.spec.ts:46]: I wonder if we did this when we first introduced it? Agreed it seems fine to remove now. BTW what was (possibly) causing the flakiness?

File: packages/functional-tests/tests/syncV3/syncV3ResetPassword.spec.ts [hunk 3]
```diff
@@ -5,94 +5,56 @@
...
+    await resetPassword.resetNewPassword(credentials.email);
 
-    //Verify the error
+    // Verify the error
     expect(await login.minLengthSuccess()).toBe(true);
     expect(await login.notEmailFailError()).toBe(true);
     expect(await login.notCommonPasswordUnmetError()).toBe(true);
+
+    await resetPassword.resetNewPassword('Newpassword@');
+    expect(await resetPassword.completeResetPasswordHeader()).toBe(true);
```
Comment (LZoog) [inline packages/functional-tests/tests/syncV3/syncV3ResetPassword.spec.ts:55]: In a recent PR I merged, I had removed the `expect...toBe(true)` when using `await`. I feel like we should verify that soon so we can stop adding those or fix places where we might need to add it 😬

**Review by LZoog — APPROVED** [review]
LGTM!

Btw, this reminds me we need a ticket filed [for this](https://mozilla.slack.com/archives/CLV3KMZ8B/p1678311867458179?thread_ts=1678311047.186009&cid=CLV3KMZ8B), I think it's going to cause the same issue next time we push to prod. Does what I suggested sound good to you? I'll try to get an issue filed before planning.

File: packages/functional-tests/tests/syncV3/signUp.spec.ts (hunk 2 above)
Comment (vbudhram) [reply packages/functional-tests/tests/syncV3/signUp.spec.ts:46]: Not really sure, I think it might be cascading issues from other tests.

**Pull request merged**